### PR TITLE
lint issues, except include Web3

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/data/data-web3.html [hunk 1]
```diff
@@ -4,6 +4,7 @@
 This code may only be used under the license found at https://github.com/swarmcity/license
 -->
 <link rel="import" href="../../bower_components/polymer/polymer-element.html">
+<link rel="import" href="./Web3.html">
 <script src="../swarm-city.min.js" async></script>
```
Comment (bkawk) [inline src/data/data-web3.html]: This import will expose swarmCity.web3

File: src/data/data-web3.html [hunk 2]
```diff
@@ -4,6 +4,7 @@
 This code may only be used under the license found at https://github.com/swarmcity/license
 -->
 <link rel="import" href="../../bower_components/polymer/polymer-element.html">
+<link rel="import" href="./Web3.html">
```
Comment (bkawk) [inline src/data/data-web3.html]: This will also reveal the web3 object

File: src/data/data-web3.html [hunk 3]
```diff
@@ -267,32 +282,33 @@
...
              */
             _getHashTagIpfsTag(contractName, lookupHashtag) {
                 return new Promise((resolve, reject) => {
-
-                    Promise.all([this._loadAbi(contractName), this._loadAddress(contractName)])
-                        .then((data) => {
-                            // constructing contract from abi/address should happen globally, not per request
-                            const abiArray = data[0].abi;
-                            const contractAddress = data[1][contractName];
-                            var web3 = new Web3();
```
Comment (bkawk) [inline src/data/data-web3.html:281]: can we use swarmCity.Web3() ?? or was there some munging in the browserify?

Comment (p-s-dev) [reply src/data/data-web3.html:281]: needed to import as Web3, call `new` and set provider

**Pull request merged**